### PR TITLE
marking keeps {guided: true} in heimdal/mixpanel if from guide

### PR DESCRIPTION
**Pull request by 2is10**
@martinraison 
https://app.asana.com/0/2456298849186/14723254070016

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/RawBookmarkRepresentation.scala [hunk 1]
```diff
@@ -5,7 +5,12 @@ import com.keepit.common.healthcheck.AirbrakeNotifier
 import play.api.libs.json.{ JsArray, JsObject, JsValue }
 import com.keepit.model.{ URLFactory, Normalization }
 
-case class RawBookmarkRepresentation(title: Option[String] = None, url: String, isPrivate: Boolean, canonical: Option[String] = None, openGraph: Option[String] = None)
+case class RawBookmarkRepresentation(
+  title: Option[String] = None,
+  url: String,
+  isPrivate: Boolean,
+  canonical: Option[String] = None,
+  openGraph: Option[String] = None)
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RawBookmarkRepresentation.scala:13]: ended up not changing this. just wrapped for readability.

**Comment by 2is10** [comment]
merging, happy to address any comments in a follow-up PR

File: packrat/main.js [hunk 2]
```diff
@@ -644,7 +644,7 @@ api.port.on({
       api.tabs.emit(tab, 'kept', {fail: true});
     } else if (!d.state) {
       d.state = 'keeping';
-      ajax('POST', '/ext/keeps', {
+      ajax('POST', '/ext/keeps' + (data.guided ? '?guided=true' : ''), {
```
Comment (martinraison) [inline packrat/main.js:647]: I may have put the `guided` parameter in the body, but I guess it's a matter of taste (and it doesn't really matter).

**Comment by martinraison** [comment]
looks good to me!

**Pull request merged**